rs/sns/governance/src/governance.rs

```diff
@@ -2545,6 +2546,33 @@ impl Governance {
             })
     }
 
+    pub fn check_no_upgrades_in_progress(&self) -> Result<(), GovernanceError> {
```

Comment:
On second thoughts (and as discussed over zoom), 

let's have just one function that takes `Option<ProposalId>`

Reply:
Done!